tamr_unify_client/models/project/mastering.py

```diff
@@ -76,4 +76,19 @@ def published_clusters(self):
         alias = self.api_path + "/publishedClusters"
         return Dataset(self.client, None, alias)
 
+    def record_clusters_with_data(self):
+        """Record clusters with data.
+
+        Call :func:`~tamr_unify_client.models.dataset.resource.Dataset.refresh` from
+        this dataset to generate clusters with data.
+
+        Function is a workaround because API is broken.
```

Comment:
This sounds a bit too dramatic. Plus the purpose of the Python Client is to shield users from having to deal with some of the sharp edges of the API (like this one). Since docstrings like this are used to generate docs read by end users, we should just remove this text.